fix double casting

PerkinsDan opened the pull request.

QMULDA reviewed (approved):
Approved - just curious abt why you've refactored Portfolio.tsx to have "const fetchPortfolio" instead of "function usePortfolio" ?

PerkinsDan commented:
> Approved - just curious abt why you've refactored Portfolio.tsx to have "const fetchPortfolio" instead of "function usePortfolio" ?

U removed the hook file for usePortfolio() but were still calling it so I removed that first from the top.

Then the standard is to have useEffect as the most-outer function which sets an async fetch function and calls it from within the useEffect() so i just updated it. but i had to declare that inner fetch function
